test/idler_assertion.go

```diff
@@ -197,6 +197,7 @@ func (a *IdleablePayloadAssertion) DeploymentConfigScaledDown(deployment *opensh
 	err := a.client.Get(context.TODO(), types.NamespacedName{Name: deployment.Name, Namespace: deployment.Namespace}, d)
 	require.NoError(a.t, err)
 	assert.Equal(a.t, int32(0), d.Spec.Replicas)
+	assert.False(a.t, d.Spec.Paused) // DeploymentConfig should be unpaused when scaling down so that the replicas update can be rolled out
```

Comment:
Shouldn't we set it to true and then check it was reset to false? Otherwise the test would pass even if we don't reset it, right?

Reply:
sure, added 956344941b7a76c9a36581f3444a3fe94773a6f8